File: app/src/main/java/br/com/ribamar/concrete/model/pojos/requests/License.java

package br.com.ribamar.concrete.model.pojos.requests;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class License {

    @Expose
    private String key;
    @Expose
    private String name;
    @SerializedName("node_id")
    private String nodeId;
    @SerializedName("spdx_id")
    private String spdxId;
    @Expose
    private String url;

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getNodeId() {
        return nodeId;
    }

    public void setNodeId(String nodeId) {
        this.nodeId = nodeId;
    }

    public String getSpdxId() {
        return spdxId;
    }

    public void setSpdxId(String spdxId) {
        this.spdxId = spdxId;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

}

File: app/src/main/java/br/com/arthur/githubapp/model/License.kt
package br.com.arthur.githubapp.model

import com.google.gson.annotations.SerializedName

class License(

    @SerializedName("key")
    val key: String,

    @SerializedName("name")
    val name: String,

    @SerializedName("node_id")
    val nodeId: String,

    @SerializedName("spdx_id")
    val spdxId: String,

    @SerializedName("url")
    val url: String

)